offchain/inspect-server/src/config.rs

```diff
@@ -15,13 +15,19 @@
 /// Command-line parameters take precedence over environment variables and environment variables
 /// take precedence over same parameter from file configuration.
 use serde::Deserialize;
-use snafu::Snafu;
+use snafu::{OptionExt, ResultExt, Snafu};
 use structopt::StructOpt;
 
 #[derive(Debug, Snafu)]
-#[snafu(display("Bad configuration: {}", err))]
-pub struct BadConfigurationError {
-    err: String,
+pub enum ConfigError {
+    #[snafu(display("Bad configuration: {}", err))]
+    BadConfigurationError { err: String },
+
+    #[snafu(display("Fail to read config file"))]
```

Comment:
Should we standardize error messages like other in other modules?
For example, `read configuration file error` and `parse configuration file error`.